### ISSUE TITLE
ROSHAN went to the BOTTOM position (light side) when the NIGHT FELL

### ISSUE THREAD

**Issue by slegka0** [ValveSoftware/Dota2-Gameplay#11959]
### Description

I attach a short video (30 sec), please watch it till the end: at the 45th minute of the game, night falls, Roshan goes to the upper side, BUT immediately returns back to the lower side, although it is night outside (whaat?)

### Example Match ID (and possibly Timestamp)

7336204556 - 44:55 - 45:25

### Screenshots

https://drive.google.com/file/d/1b3yicmSZyQeW8kt9mXGlT-IhpcF8u3-h/view?usp=sharing

**Comment by TobyFlendersonFromHR** [comment]
That's correct, Nightfall is a pro player who is currently playing for BetBoom Team.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/11987
